config/registry/aws/modules/cloudfront-distribution.yaml

```diff
@@ -0,0 +1,60 @@
+halt: false
+environment_module: false
+terraform_module: aws-cloudfront-distribution
```

Comment:
Can we remove the `aws-` from here too?

Reply:
preferably no-- right now the terraform modules are all stored in a single subdirectory (to be changed soon with module api), and so the cloud prefixes actually started a a means of keeping them somewhat sorted. For all intents and purposes the module as seen by our code and the user is cloudfront-distribution but its terraform module directory happens to be aws-cloudfront-distribution

Reply:
Ah cool. If there is no user visible component, then it's fine!